code/modules/mob/living/simple_mob/subtypes/vore/dragon.dm

```diff
@@ -34,22 +34,22 @@
 	ai_holder_type = /datum/ai_holder/simple_mob/melee
 	say_list_type = /datum/say_list/dragonboss
 
-/mob/living/simple_mob/vore/dragon/Process_Spacemove(var/check_drift = 0)
+/mob/living/simple_mob/vore/aggresive/dragon/Process_Spacemove(var/check_drift = 0)
 	return 1	//No drifting in space for space dragons!
 /*
-/mob/living/simple_mob/vore/dragon/FindTarget()
+/mob/living/simple_mob/vore/aggresive/dragon/FindTarget()
```

Comment:
This can be made into a 
/datum/say_list/dragon
     say_got_target = list("snaps at [H]") or whatever the target mob letter is

Also aggressive

Reply:
no memes + out of scope